### PR TITLE
Project 3 - Ati Takrami

### PR DESCRIPTION
**Pull request by takrami**
This is my website for week 3.

### CODE REVIEW

File: code/assets/css/animations.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@keyframes shake {
```
Comment (EmOlsz) [inline code/assets/css/animations.css:1]: You've implemented impressive animations, great job :)

File: code/assets/css/footer.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  font-size: var(--large);
+  font-weight: 600;
+  display: inline-block;
+}
+
+.end-footer:hover a {
+  animation-name: vibration;
+  animation-duration: 0.8s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
```
Comment (EmOlsz) [inline code/assets/css/footer.css:52]: As a side note - we can always use animation shorthand :

```
animation: vibration 0.8s linear infinite;
```

File: code/assets/css/form.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  margin-top: calc(var(--spacing) * 2);
+}
+
+input:not([type="radio"]):not([type="checkbox"]),
+textarea {
+  display: block;
+  margin-bottom: calc(var(--spacing) * 4);
+  width: 100%;
+  padding: calc(var(--spacing) * 2);
+  /* border: 1px solid red; */
```
Comment (EmOlsz) [inline code/assets/css/form.css:43]: Before final push always check your files and remove all commented code - needs to be fixed

File: code/assets/css/header.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
+.header-video {
+  /* object-fit: cover; */
```
Comment (EmOlsz) [inline code/assets/css/header.css:2]: And over here

File: code/assets/css/homepage.css [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+}
+
+/* DESKTOP */
+@media only screen and (min-width: 980px) {
+  .transform-sec {
+    transform: translateY(-130px);
+  }
+
+  .accordion {
+    /* background-color: var(--primaryColorLight); */
```
Comment (EmOlsz) [inline code/assets/css/homepage.css:101]: and here ;)

File: code/assets/css/homepage.css [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+  }
+
+  .accordion-inner {
+    flex: 0.4;
+  }
+
+  .faq-container {
+    background-color: var(--white);
+    /*  box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px; */
+    /* border-radius: var(--spacing); */
```
Comment (EmOlsz) [inline code/assets/css/homepage.css:125]: comments

File: code/index.html [hunk 7]
```diff
@@ -1,30 +1,276 @@
...
+        />
+
+        <source
+          src="./assets/video/SampleVideo_360x240_1mb.mp4"
+          type="video/mp4"
+          media="(max-width: 600px)"
+        />
+        <source
+          src="./assets/video/SampleVideo_360x240_1mb.webm"
+          type="video/webm"
```
Comment (EmOlsz) [inline code/index.html:49]: Very nice code formatting!

File: code/index.html [hunk 8]
```diff
@@ -1,30 +1,276 @@
...
+          media="(max-width: 600px)"
+        />
+
+        <p>This browser doesn't support video...</p>
+      </video>
+      <!-- LOGO -->
+      <a href="/" class="block logo"
+        ><img src="./assets/img/logo.png" alt=""
+      /></a>
```
Comment (EmOlsz) [inline code/index.html:58]: Ups, let's fix formatting here

File: code/index.html [hunk 9]
```diff
@@ -1,30 +1,276 @@
...
+          <h2>
+            Explore more about climbing and how you can be a professional
+            climber.
+          </h2>
+        </header>
+        <ul class="first-box-list">
+          <li class="first-box-items">
+            <a href="#" class="block">
+              <figure>
+                <img src="./assets/img/02.webp" alt="" class="res-img" />
```
Comment (EmOlsz) [inline code/index.html:86]: There is really no point in setting `alt` attribute as empty string, let's add some meaningful message there

File: code/script.js [hunk 10]
```diff
@@ -1,9 +1,35 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+function transformSection() {
```
Comment (EmOlsz) [inline code/script.js:1]: Impressive once again! :)

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hey Ati!
Great project, I'm truly happy to see such advanced and well done code! You did some pro development this week :)
There are some small things I would like you to fix (commented code and `alt` tag), take a look at my comments and provide changes.
Moreover, heck out [Week 3 Master document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475)

**Comment by takrami** [comment]
Thank you! :) I've done the changes.

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**